core/src/test/java/org/elasticsearch/action/admin/cluster/snapshots/SnapshotBlocksTests.java

```diff
@@ -126,17 +126,13 @@ public void testCreateSnapshotWithIndexBlocks() {
 
     @Test
     public void testDeleteSnapshotWithBlocks() {
-        logger.info("-->  deleting a snapshot is blocked when the cluster is read only");
+        logger.info("-->  deleting a snapshot is allowed when the cluster is read only");
```

Comment:
I understand why we would allow creating a snapshot when the cluster is read-only, but it seems backwards to allow deleting a snapshot, should we document this somewhere in the snapshot & restore documentation?